Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -1052,6 +1075,16 @@ void ACesium3DTileset::LoadTileset() {
     this->BoundingVolumePoolComponent->initPool(this->OcclusionPoolSize);
   }
 
+  TSharedPtr<CesiumGeospatial::Ellipsoid> pNativeEllipsoid =
+      this->ResolveGeoreference()->GetEllipsoid()->GetNativeEllipsoid();
+
+  // Since native will be working with an STL shared_ptr, not an Unreal one,
+  // we'll copy the ellipsoid here. It's not much data to copy and saves us some
+  // headaches.
+  std::shared_ptr<CesiumGeospatial::Ellipsoid> pEllipsoid =
+      std::make_shared<CesiumGeospatial::Ellipsoid>(
+          pNativeEllipsoid->getRadii());
```

Comment:
If my memory serves me, cesium-native no longer uses shared pointers for its internal use of ellipsoid.

So we shouldn't need this logic, and `GetNativeEllipsoid` can probably just return a `CesiumGeospatial::Ellipsoid` rather than a share ptr to one.